benchmarks/src/main/scala/benchmarks/Main.scala

```diff
@@ -6,10 +6,13 @@ object Main {
   def main(args: Array[String]): Unit = {
     val benchmarks = Discover.discovered
 
+    val format = args lift (0) map (Format(_)) getOrElse TextFormat
```

Comment:
Lets put a bit more dots in there. Infix is fine for symbolic operators but for text ones it's a bit overkill.